src/modules/app/app.service.ts

```diff
@@ -57,21 +59,48 @@ export class AppService {
     createRoom (roomName: string, isPrivate: boolean) {
         return this.roomRepository.save({
             roomName,
-            isPrivate
+            isPrivate,
+            messages: []
         })
     }
 
-    createMessage (messageContent: string, isImage: boolean, date: Date, userId: string, roomName: string) {
+    async createMessage (messageContent: string, isImage: boolean, date: Date, userId: string, roomName: string) {
+        const currentRoom = await this.roomRepository.findOne({where: { roomName } })
+
         return this.messageRepository.save({
             messageContent,
             isImage,
             date,
             socketId: userId,
-            roomName
+            room: currentRoom
+        })
+    }
+
+    createUserRoom (socketId: string, roomName: string) {
+        return this.userRoomRepository.save({
+            roomName,
+            socketId,
+            createdAt: new Date(),
+            updatedAt: new Date()
         })
     }
 
+    async addUserToRoom (roomName: string, socketId: string) {
+        const userRoom = await this.userRoomRepository.findOne({ where: { roomName, socketId } })
```

Comment:
enter between consts and ifs. Group assignments together and instructions together